website/docs/components/button-set/partials/code/how-to-use.md

```diff
@@ -9,17 +9,57 @@ The basic invocation requires two or more buttons to be provided as children:
 </Hds::ButtonSet>
 ```
 
+Other button-like components can also be used as children such as the `Dropdown`.
```

Comment:
I could add other examples if others have ideas.

Reply:
Could be useful to add a bit of that and/or reference the Button pattern docs. I'll take a look...